packages/templates/tests.spec.ts

```diff
@@ -55,46 +55,48 @@ describe("Templates", () => {
   // for each project + language
   for (const projectType of Object.keys(projectLanguages)) {
     for (const language of projectLanguages[projectType]) {
-      // run the test commands at commands[language]
-      let commands = languageTestCommands[language];
+      describe(`${projectType}/${language}`, () => {
+        // run the test commands at commands[language]
+        let commands = languageTestCommands[language];
```

Comment:
sorry for being too pedantic but why don't we have `for...describe...for...describe` instead

Reply:
My reasoning is that it would be much easier to execute the selected tests if we have proper hierarchy. 
`jest -t "projectType"` would execute every project type and `jest -t "projectType language"` would execute project with the specified language

Reply:
Heheh, it's alright, and we can go with that!